docs/modules/ROOT/pages/api/access.adoc

```diff
@@ -0,0 +1,477 @@
+* xref:#AccessControl-assert_only_role[`++assert_only_role(self, role)++`]
+--
+
+[.contract-index]
+.Events
+--
+.IAccessControl
+* xref:#AccessControl-RoleAdminChanged[`++RoleAdminChanged(role, previous_admin_role, new_admin_role)++`]
+* xref:#AccessControl-RoleGranted[`++RoleGranted(role, account, sender)++`]
+* xref:#AccessControl-RoleRevoked[`++RoleRevoked(role, account, sender)++`]
+--
+
+[#AccessControl-External-Functions]
+==== External Functions
+
+[.contract-item]
+[[AccessControl-has_role]]
+==== `[.contract-item-name]#++has_role++#++(self: @ContractState, role: felt252, account: ContractAddress) → bool++` [.item-kind]#external#
+
+Returns `true` if `account` has been granted `role`.
+
+[.contract-item]
+[[AccessControl-get_role_admin]]
+==== `[.contract-item-name]#++get_role_admin++#++(self: @ContractState, role: felt252) → felt252++` [.item-kind]#external#
+
+Returns the admin role that controls `role`. See {grant_role} and
+{revoke_role}.
+
+To change a role's admin, use {_set_role_admin}.
+
+[.contract-item]
+[[AccessControl-grant_role]]
+==== `[.contract-item-name]#++grant_role++#++(ref self: ContractState, role: felt252, account: ContractAddress)++` [.item-kind]#external#
+
+Grants `role` to `account`.
+
+If `account` had not been already granted `role`, emits a {RoleGranted}
+event.
+
+Requirements:
+
+- the caller must have ``role``'s admin role.
+
+May emit a {RoleGranted} event.
+
+[.contract-item]
+[[AccessControl-revoke_role]]
+==== `[.contract-item-name]#++revoke_role++#++(ref self: ContractState, role: felt252, account: ContractAddress)++` [.item-kind]#external#
+
+Revokes `role` from `account`.
+
+If `account` had been granted `role`, emits a {RoleRevoked} event.
+
+Requirements:
+
+- the caller must have ``role``'s admin role.
+
+May emit a {RoleRevoked} event.
+
+[.contract-item]
+[[AccessControl-renounce_role]]
+==== `[.contract-item-name]#++renounce_role++#++(ref self: ContractState, role: felt252, account: ContractAddress)++` [.item-kind]#external#
+
+Revokes `role` from the calling account.
+
+Roles are often managed via {grant_role} and {revoke_role}. This function's
+purpose is to provide a mechanism for accounts to lose their privileges
+if they are compromised (such as when a trusted device is misplaced).
+
+If the calling account had been revoked `role`, emits a {RoleRevoked}
+event.
+
+Requirements:
+
+- the caller must be `account`.
+
+May emit a {RoleRevoked} event.
+
+[.contract-item]
+[[AccessControl-supports_interface]]
+==== `[.contract-item-name]#++supports_interface++#++(self: @ContractState, interface_id: felt252) → bool++` [.item-kind]#external#
+
+Returns whether a contract implements a given interface or not.
+
+[#AccessControl-Internal-Functions]
+==== Internal Functions
+
+[.contract-item]
+[[AccessControl-initializer]]
+==== `[.contract-item-name]#++initializer++#++(ref self: ContractState)++` [.item-kind]#internal#
+
+Initializes the contract by registering the IAccessControl interface ID.
```

Comment:
What do you think about linking `IAccessControl interface ID` to the `IAccessControl` section? This way, users will be able to find the interface id when they click on the link